src/hooks/use-focus.ts

```diff
@@ -29,29 +39,35 @@ interface Output {
  * This hook returns an object with `isFocused` boolean property, which
  * determines if this component is focused or not.
  */
-const useFocus = ({isActive = true, autoFocus = false}: Input = {}): Output => {
+const useFocus = ({
+	isActive = true,
+	autoFocus = false,
+	id
+}: Input = {}): Output => {
 	const {isRawModeSupported, setRawMode} = useStdin();
-	const {activeId, add, remove, activate, deactivate} = useContext(
+	const {activeId, add, remove, activate, deactivate, focus} = useContext(
 		FocusContext
 	);
 
-	const id = useMemo(() => Math.random().toString().slice(2, 7), []);
+	const resolvedId = useMemo(() => id ?? Math.random().toString().slice(2, 7), [
```

Comment:
Could you keep this variable name named as `id`, and rename `id` in arguments to `customId` as such?

```js
const useFocus = ({ id: customId }) => ...
```

Reply:
Done!